Topics/System_Design/Mobile_App_System_Design.md

```diff
@@ -0,0 +1,33 @@
+# Essential Elements of Mobile App System Design
+
+## Introduction
+For the last 15 years, mobile apps have emerged as vital tools in our daily lives. The best mobile apps simplify our lives by helping us with daily tasks. What lies behind these apps is a thoughtful system design that has important considerations that differ from traditional desktop devices and must be taken into account when mobile app system design, namely: UI and Performance.
+
+
+## User Interface Design
+
+### 1. Screen Size
+Compared to desktops, mobile screens are much smaller. The smaller screen size requires specific design considerations, in particular, mobile apps should prioritize essential features and focus on presenting relevant content. For example, in an e-commerce mobile app, the focus should be on the products being sold, in particular, the images of the products with more detailed information being hidden but still seamlessly accessible.
+
+### 2. Touch Interface
+The touch interface of mobile devices should be also kept in mind when designing mobile applications. Unlike mouse clicks, touch interactions are not as precise, requiring designing larger and more accessible touch targets. In the example of an e-commerce mobile app, the sold products should all have a large image that is tapable and clearly separated from the other products. A Sub consideration of the touch interface is also that most users operate their devices with one hand and therefore tap using the thumb. Therefore any important elements to an application should be at the bottom of the screen. For example, on Instagram, the navigation bar is at the bottom of the screen so it is easily accessible to the thumb of the user.
```

Comment:
Are there exceptions to this case? I'm sure we have all seen bad UI design or buttons that are incredibly small. Maybe you could talk about scenarios in which this may happen, or show images of bad touch interfaces.

Reply:
I would say there might be; however, it is such an edge case that I don't think it's worth including it.